IPython/core/shellapp.py

```diff
@@ -100,7 +100,7 @@
 
 class InteractiveShellApp(Configurable):
     """A Mixin for applications that start InteractiveShell instances.
-    
+
```

Comment:
No worries - Atom's behaviour is a sensible default for new files, but a bit annoying for big codebases like this with trailing whitespace already there. We'll live with it for this PR.